armi/reactor/blocks.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, name: str, height: float = 1.0):
 
         self.points = []
         self.macros = None
+        self.axialExpTargetComponent = None
```

Comment:
It might be better if this was a parameter that was a string. This will (1) keep attributes stored a block to a minimum and (2) allow target components can be determined on DB loading, (3) allow us to make sure that we can track changes to the parameters as time goes on (code changes) and also will allow us to visually verify the target expansion component with a DB viewer.

Reply:
We are going open a separate issue/PR on this!